app/webpacker/components/RegistrationsV2/Register/RegistrationOverview.jsx

```diff
@@ -107,7 +107,7 @@ export default function RegistrationOverview({
                 {I18n.t('activerecord.attributes.registration.comments')}
                 :
               </List.Header>
-              {registration.competing.comment.length > 0 ? registration.competing.comment : I18n.t('competitions.schedule.rooms_panel.none')}
+              {registration.competing.comment?.length > 0 ? registration.competing.comment : I18n.t('competitions.schedule.rooms_panel.none')}
```

Comment:
This frontend change is necessary because we're adding `strip_attributes` to the backend in this PR, and `strip_attributes` turns "blank-only" values into `nil` in Ruby (which gets serialized as `null` to the frontend)